Bank/src/main/java/com/ark/bank/BankController.java

```diff
@@ -463,4 +463,18 @@ public void update(Observable o, Object arg) {
             notifyObservers(arg);
         }
     }
+
+    @Override
+    public boolean changeCreditLimit(String sessionKey, BankAccount bankAccount, long limit) {
```

Comment:
Waarom een andere naam?  Noem het gewoon setCreditLimit(String.....). Dat is makkelijker, want dan weet je meteen waar het bij hoort.